src/styles/variables.css

```diff
@@ -43,6 +44,9 @@
   --temp-grey-blue-6: rgb(231, 242, 246); /* Used for user profile gradient end color */
   --temp-grey-blue-7: rgb(118, 116, 139); /* Used for the wallet connect button in the user navigation */
   --drop-shadow: rgba(62, 118, 244, 0.14);
+  --light-border: rgba(54, 54, 54, 0.15);
```

Comment:
Please don't add rgba colors in.

Alpha transparency should be achieved with `color-mod`